docs/mine/lotus/split-markets-miners.md

```diff
@@ -267,7 +267,11 @@ your node.
 lotus-miner actor set-addrs <NEW_MULTIADDR>
 ```
 
-#### Step 4. **Optional**: Move the DAG store directory to the markets node repository
+#### Step 4. Copy Existing Storage Locations
+
+If you have [customized storage location](https://docs.filecoin.io/mine/lotus/miner-lifecycle/#changing-storage-locations) for your monolith miner, we **highly recommend** you copy `~/.lotusminer/storage.json` to the `~/.lotusmarekt` as well. By doing this, when requests come in to the markets node, the markets node will read the data directly from disk instead of going through the miner node. 
```

Comment:
I wouldn't recommend copying blindlessly because these processes may be running in different machines.